4/4ti2/build_tarballs.jl

```diff
@@ -2,7 +2,7 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder
 
-name = "4ti2"
+name = "_4ti2"
```

Comment:
I'm not really a big fan of this option... Note that in any case the directory name must exactly match this name

Reply:
Yeah I just discovered this -- the tarballs it creates are named _4ti2FOO.tar.gz but it tries to upload 4ti2FOO.tar.gz

So, what alternative would you suggest?

Reply:
@ararslan suggested `lib4ti2`.  We usually try to use the same name as the project, rather than one of the products, but this is pathological case and I think that `lib4ti2` would work better

Reply:
Renamed. Note that there is still this:
```
   ExecutableProduct("4ti2gmp", :_4ti2gmp),
   ExecutableProduct("4ti2int32", :_4ti2int32),
   ExecutableProduct("4ti2int64", :_4ti2int64),
```
Of course I could also replace the `_` there with `lib`, but I fear this will be very confusing, given that there is also this:
```
   LibraryProduct("lib4ti2gmp", :lib4ti2),
   LibraryProduct("lib4ti2int32", :lib4ti2int32),
   LibraryProduct("lib4ti2int64", :lib4ti2int64),
```